protocols/protocol/cloudstate/crdt.proto

```diff
@@ -92,8 +118,20 @@ message CrdtStreamIn {
     }
```

Comment:
Why is the `CrdtDelta` message named _changed_ and not _delta_?

Reply:
@marcellanz Good question. Arguably it should then be `change` rather than `changed` in any case. :) I have no real preference here, especially as names shouldn't matter in proto :D

Reply:
@viktorklang There could be a good reason in the domain of CRDTs which I don't know well. I have no preference too, but would expect to have them named consistent, as the other ones are.
```
CrdtInit init = 1;
CrdtState state = 2;
CrdtDelta changed = 3;
CrdtDelete deleted = 4;
Command command = 5;
```